.github/workflows/release.yml

```diff
@@ -6,7 +6,7 @@ on:
       - "v*"
 
 env:
-  SETUP_GO_VERSION: '1.18'
+  SETUP_GO_VERSION: '1.20'
```

Comment:
This is just for something release related correct, and won't apply to the build of the ui-backend? That's currently still 1.18 - https://github.com/epinio/ui-backend/blob/main/src/jetstream/go.mod#LL3C1-L3C8

Reply:
> This is just for something release related correct, and won't apply to the build of the ui-backend? That's currently still 1.18 - https://github.com/epinio/ui-backend/blob/main/src/jetstream/go.mod#LL3C1-L3C8

@richard-cox I am not sure. The idea was to be in sync with this other [pr](https://github.com/epinio/ui-backend/pull/30/files#diff-e0cfdbd18ccf2a8c9fb50fd963ecc670fb85a9cce968960b31d475b0d0263a1fR14) for the Release Next workflow. @enrichman do you foresee an impact on the build of the ui-backend? If unsure I can revert back to `1.18`

Reply:
maybe the ui-backend needs to be set to 1.20 to be consistent with all the other places (?)

Reply:
thanks. I just returned it back to 1.18 for the moment. Perhaps we can re-take this conversation in a separate pr.